### DISCUSSION TITLE
Turn Lights on/off

### DISCUSSION THREAD

**Discussion by Webfeger** [andycb/AdventurerClientJS#73] [Ideas]
Here are the Codes, to turn Lights on or off:

M146
LED control.

LED on command
M146 r255 g255 b255 F0
LED off command
M146 r0 g0 b0 F0

RGB needs to be lowercase. In addition, although the value of rgb was changed to something other than 0 or 255 before, there was no change in the actual color of the LED. It may only recognize display and erase.

It's possible to add this commands in this application? I never wrote applications in electron/angular so i still figure out how this application works.

**Comment by andycb** [comment]
Hey,

Thanks for the printer commands for the lights. If you want an example of adding a new command to the app - take a look in #39. There's also a mock up of what the button might look like in #43.
